### PR TITLE
build-x86-images: rewrite

### PR DESCRIPTION
**Pull request by paper42**
* rename e to enlightenment
* enable building kde by default
* allow building multiple, but not all images (ex. -b "base xfce")
* run the script with set -e (exit when there is an error)

### CODE REVIEW

File: build-x86-images.sh.in [hunk 1]
```diff
@@ -1,90 +1,58 @@
...
+    fi
+    IMG=void-live-${ARCH}-${DATE}${IMG_SUFFIX}.iso
+    GRUB_PKGS="grub-i386-efi grub-x86_64-efi"
+    PKGS="dialog cryptsetup lvm2 mdadm void-docs-browse $GRUB_PKGS"
+
+    if [ "$variant" != base ]; then
+        PKGS="$PKGS xorg-minimal xorg-input-drivers xorg-video-drivers setxkbmap xauth font-misc-misc terminus-font dejavu-fonts-ttf alsa-plugins-pulseaudio"
+    fi
+
+    if [ "$variant" = enlightenment ]; then
```
Comment (classabbyamp) [inline build-x86-images.sh.in:34]: this might be cleaner as a `case`?

File: build-x86-images.sh.in [hunk 2]
```diff
@@ -1,90 +1,75 @@
...
+            PKGS="$PKGS $XORG_PKGS gnome firefox-esr"
+        ;;
+        kde)
+            PKGS="$PKGS $XORG_PKGS kde5 konsole firefox-esr dolphin"
+        ;;
+        lxde)
+            PKGS="$PKGS $XORG_PKGS lxde lxdm gvfs-afc gvfs-mtp gvfs-smb udisks2 firefox-esr"
+        ;;
+        lxqt)
+            PKGS="$PKGS $XORG_PKGS lxqt lxqt gvfs-afc gvfs-mtp gvfs-smb udisks2 qupzilla"
```
Comment (CameronNemo) [inline build-x86-images.sh.in:60]: you put lxqt twice

**Review by classabbyamp — APPROVED** [review]
code looks good

File: build-x86-images.sh.in (hunk 2 above)
Comment (paper42) [reply build-x86-images.sh.in]: oops, fixed

**Pull request merged**